android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -487,6 +506,14 @@ class DVCClient private constructor(
             return this
         }
 
+        private fun getAnonUserId(): String? {
+            return dvcSharedPrefs?.getString(DVCSharedPrefs.AnonUserIdKey)
+        }
+
+        private fun setAnonUserId(anonId: String) {
+            dvcSharedPrefs?.saveString(anonId, DVCSharedPrefs.AnonUserIdKey)
+        }
+
```

Comment:
could move this to `SharedPrefs` to not have duplicate code

Reply:
these wrappers aren't doing much, we could just use the dvcSharedPrefs getString/saveString methods directly, or move them into DVCSharedPrefs — your call